src/NHibernate.Test/Async/EntityModeTest/Map/Basic/DynamicClassFixture.cs

```diff
@@ -178,6 +190,55 @@ protected override IList Mappings
 				await (t.CommitAsync(cancellationToken));
 			}
 		}
+/*
```

Comment:
Why is this commented?

Reply:
I forgot to regen async code after some fiddling for trying to dodge the sync generation crash.

Reply:
In fact I cannot regenerate it^^. I have to revert the change.